cmake/src/main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "../include/hello_world.h"
```

Comment:
As for hello_world.cpp. change include style

Reply:
+